docs/source/contribution.rst

```diff
@@ -3,197 +3,64 @@
+   If you have a question regarding the usage of Chainer, it is recommended to send a post to `StackOverflow <https://stackoverflow.com/>`_ or the `Chainer User Group <https://groups.google.com/forum/#!forum/chainer>`_ instead of the issue tracker.
    The issue tracker is not a place to share knowledge on practices.
-   We may suggest these places and immediately close how-to question issues.
 
 How to Send a Pull Request
 ~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-If you can write code to fix an issue, we encourage to send a PR.
+If you can write code to fix an issue, it is encouraged to send a PR.
 
-First of all, before starting to write any code, do not forget to confirm the following points.
+In that case, confirm the following points before starting to write any code.
 
-- Read through the :ref:`coding-guide` and :ref:`testing-guide`.
-- Check the appropriate branch that you should send the PR following :ref:`contrib-git-branches`.
-  If you do not have any idea about selecting a branch, please choose the ``master`` branch.
-
-In particular, **check the branch before writing any code.**
-The current source tree of the chosen branch is the starting point of your change.
+- Read the :ref:`coding-guide` and the :ref:`testing-guide`.
+- Check the appropriate branch to which you should send a PR, following :ref:`contrib-git-branches`.
+  If you are unsure about which branch to target, choose the ``master`` branch.
+  The current source tree of the chosen branch is the starting point of your change.
 
 After writing your code **(including unit tests and hopefully documentations!)**, send a PR on GitHub.
-You have to write a precise explanation of **what** and **how** you fix;
-it is the first documentation of your code that developers read, which is a very important part of your PR.
+You have to write a precise explanation of **what** and **how** in the description;
+this is the first documentation of your code and an important part of your PR.
 
-Once you send a PR, it is automatically tested on `Travis CI <https://travis-ci.org/chainer/chainer/>`_ for Linux and Mac OS X, and on `AppVeyor <https://ci.appveyor.com/project/chainer/chainer>`_ for Windows.
-Your PR needs to pass at least the test for Linux on Travis CI.
-After the automatic test passes, some of the core developers will start reviewing your code.
-Note that this automatic PR test only includes CPU tests.
+However, even if your code is not complete, you can send a PR as a *work-in-progress (WIP) PR* by prefixing the PR title with ``[WIP]``.
+If you just describe the PR, the core team and other contributors can join the discussion about how to proceed with it.
+WIP PRs may occasionally be be useful for discussing based on concrete code.
 
-.. note::
+When a PR is created (or updated), it is automatically tested in the CI environment.
```

Comment:
Is it about Travis CI? If so, the next line is inaccurate.

Reply:
Clarified it a bit. Can you take a look?

Reply:
Don't we need to mention triggering of other CIs? (just writing "manually by a committer" might be ok)

Reply:
Updated again, PTAL.